packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -50,7 +52,8 @@
     "vscode": "1.1.17"
   },
   "extensionDependencies": [
-    "dbaeumer.vscode-eslint"
+    "dbaeumer.vscode-eslint",
+    "salesforce.salesforcedx-vscode-core"
```

Comment:
Any considerations for `salesforcedx-vscode-lightning` and `salesforcedx-vscode-lwc` using `waitForDX` and others including `salesforce.salesforcedx-vscode-core` as extension dependency? cc @nrkruk

Reply:
I'd prefer that we didn't add a dependency on core. Why did you need to add this?

Reply:
@nrkruk I'd like to use some modules/functions from `sfdxCoreExports`, for example: `SfdxCommandlet`, `SfdxCommandletExecutor`, `telemetryService`, `notificationService`, similar to the `salesforce-vscode-apex` extension. If `extensionDependencies` is not defined prior to using the exports it errors out. It looks to me that `waitForDX` implies that the extension implicitly (and asynchronously) depends on the core extension. I could use `waitForDX` to asynchronously define the stuff that depends on `sfdxCoreExports`, but I wonder if that's necessary.